src/pam.rs

```diff
@@ -98,6 +106,47 @@ impl PamLibExt for Pam {
         let pointer = get_item(self.0, PamItemType::RUSER)?;
         unsafe { Ok(pointer.map(|p| CStr::from_ptr(p as *const c_char))) }
     }
+
+    fn conv(&self, prompt: Option<&str>, style: PamMsgStyle) -> PamResult<Option<&CStr>> {
+        let pointer = match get_item(self.0, PamItemType::CONV) {
```

Comment:
You can use ```match get_item(self.0, PamItemType::CONV)?``` and only deal with the option in the match

Reply:
Is there a cleaner way to early return like `try!`/`?` for this `Option`? Nothing I tried (mostly a bunch of chained `map` calls) made it look any better than the `match`.

Reply:
I don't know if there's a simpler way, but it looks quite good to me as it is :+1: